android/apollo/src/com/andrew/apollo/MusicPlaybackService.java

```diff
@@ -1410,15 +1418,18 @@ private void notifyChange(final String what) {
      * @param what The broadcast
      */
     private void updateRemoteControlClient(final String what) {
+
         int playState = mIsSupposedToBePlaying
-                ? RemoteControlClient.PLAYSTATE_PLAYING
+            ? RemoteControlClient.PLAYSTATE_PLAYING
+            : what.equals(PLAYSTATE_STOPPED)
+                ? RemoteControlClient.PLAYSTATE_STOPPED
                 : RemoteControlClient.PLAYSTATE_PAUSED;
```

Comment:
Here need to accurately indicate the status. Therefore use a new status PLAYSTATE_STOPPED.